common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2503,6 +2497,24 @@
     },
     "sqlState" : "42K0E"
   },
+  "INVALID_LOCATION" : {
+    "message" : [
+      "The <locationKey> location <locationValue> is invalid."
+    ],
+    "subClass" : {
+      "EMPTY" : {
+        "message" : [
+          "It cannot be empty string."
+        ]
+      },
+      "SYNTAX" : {
+        "message" : [
+          "Failed to parse URI due to <cause>"
```

Comment:
This can be a top-level error condition with 4 sub-classes as well.

Reply:
BTW, we don't need to put cause here. The java exception's `causedBy` field can carry it.

Reply:
Will address this